django/thunderstore/frontend/api/experimental/tests/test_community_package_list.py

```diff
@@ -419,12 +419,14 @@ def test_pagination(api_client: APIClient) -> None:
     data = __query_api(api_client, site.community.identifier)
 
     assert len(data["packages"]) == 24
+    assert data["has_more_pages"]
 
     page1_packages = [p["package_name"] for p in data["packages"]]
     data = __query_api(api_client, site.community.identifier, "page=2")
 
     assert len(data["packages"]) == 1
     assert data["packages"][0]["package_name"] not in page1_packages
+    assert not data["has_more_pages"]
```

Comment:
Would the `ParseError` in `paginate` be tested if you added something like;
```
with pytest.raises(ParseError):
    __query_api(api_client, site.community.identifier, "page=3")
```
?

Reply:
Well no, since DRF catches the error and returns a proper response instead. I added a separate [test case for that](https://github.com/thunderstore-io/Thunderstore/pull/552/commits/e4a11dd0cf08a18905fd020d0b155c7622cb4b63#diff-15ac0f5618a6efabdb9740435542d09b9a716ebf3319d365f9fff9c0ab927c26R433)

Reply:
Marking this resolved by myself so I can merge this since I already went ahead and merged the UI PR which depends on this change. No hard feelings @Oksamies